src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const FULLCOST = days * 40;
```

Comment:
use camel case for naming

Reply:
> use camel case for naming

i read that constants are written in uppercase and variables that are declared with let - in camelcase. is it correct to write in camelcase anyway?